models/keyword_test.go

```diff
@@ -94,15 +94,47 @@ var _ = Describe("Keyword", func() {
 	})
 
 	Describe("#GetKeywords", func() {
-		It("returns user keywords", func() {
-			user := FabricateUser("John", "john@example.com", "secret")
-			FabricateKeyword("Buy domain", false, &user)
-			FabricateKeyword("Buy bike", false, &user)
-			user2 := FabricateUser("David", "david@example.com", "secret")
-			FabricateKeyword("Buy car", false, &user2)
+		Context("given the keyword exists in the database", func() {
```

Comment:
I think this should be a new test case because `GetKeywords` function has a filtering feature. The test case you removed is the case that tests no filtering scenario and this one is the case that tests the filtering scenario.

Also, I think we should add more detail about this test case something like this
```go
Context("given keyword filter")
```
Without this detail, it is difficult to understand this test case.

For the test case with no filter, you can have this kind of context
```go
Context("given NO keyword filter")
```

What do you think?

Reply:
Ya, it makes sense